pkg/virt-controller/services/template.go

```diff
@@ -1166,9 +1166,9 @@ func wrapGuestAgentPingWithVirtProbe(vmi *v1.VirtualMachineInstance, probe *k8sv
 	return
 }
 
-func alignPodMultiCategorySecurity(pod *k8sv1.Pod, vmi *v1.VirtualMachineInstance, selinuxType string, dockerSELinuxMCSWorkaround bool) {
+func alignPodMultiCategorySecurity(pod *k8sv1.Pod, vmi *v1.VirtualMachineInstance, selinuxType string, dockerSELinuxMCSWorkaround bool, customPolicyDisabled bool) {
 	if selinuxType == "" {
-		if util.IsPasstVMI(vmi) {
+		if !customPolicyDisabled && util.IsPasstVMI(vmi) {
```

Comment:
Supposedly, if you define the FG you know what you're doing... The point of the FG is for the admin to inform KubeVirt that either they run a recent distro or that they forfeit the ability to run passt VMIs. I could add an info, like "Running a passt VMI without a custom SELinux policy, this could fail if <the FG> was set by mistake", but I'm not sure it's useful...
In general, KubeVirt is at the mercy of the node's OS. If they run a super old version of container-selinux, things will start failing left and right with cryptic errors too!

Reply:
I guess you're right, thank you.